APIHealthMonitorApplication/Logs/Log-20200723.txt

```diff
@@ -0,0 +1,57 @@
+   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
+   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
+   at APIHealthMonitorApplication.Middleware.CustomExceptionMiddleware.InvokeAsync(HttpContext httpContext) in C:\Users\mitek\Downloads\API-Health-Monitor\API-Health-Monitor\APIHealthMonitorApplication\Middleware\CustomExceptionMiddleware.cs:line 30
+2020-07-23T02:21:23.0289366+03:00 0HM1EINO1I3F2:00000001 [ERR]  (00000000)
+System.Exception: Manually generated exception in WeatherForecast Controller
+   at APIHealthMonitorApplication.Controllers.WeatherForecastController.GenerateError() in C:\Users\mitek\Downloads\API-Health-Monitor\API-Health-Monitor\APIHealthMonitorApplication\Controllers\WeatherForecastController.cs:line 42
+   at Microsoft.Extensions.Internal.ObjectMethodExecutor.<>c__DisplayClass33_0.<WrapVoidMethod>b__0(Object target, Object[] parameters)
+   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
+   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.VoidResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
+   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
+   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
+   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
+--- End of stack trace from previous location where exception was thrown ---
+   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
+   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
+   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
+--- End of stack trace from previous location where exception was thrown ---
+   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
+   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
+   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
+   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
+   at APIHealthMonitorApplication.Middleware.CustomExceptionMiddleware.InvokeAsync(HttpContext httpContext) in C:\Users\mitek\Downloads\API-Health-Monitor\API-Health-Monitor\APIHealthMonitorApplication\Middleware\CustomExceptionMiddleware.cs:line 27
```

Comment:
Please add this file to ignore list since it must be generated for each developer separately once error appeared and no Log file year

Reply:
Please remove this file from changes